waiter/test/waiter/scaling_test.clj

```diff
@@ -650,6 +650,14 @@
 (deftest scale-services-test
   (let [config {"min-instances" 1
                 "max-instances" 10}
+        service-id->scaling-states-atom (atom {})
+        update-service-scale-state! (fn [service-id scaling-state]
+                                      (swap! service-id->scaling-states-atom
+                                             (fn [service-id->scaling-states]
+                                               (if (contains? service-id->scaling-states service-id)
```

Comment:
`(update service-id->scaling-states service-id (fn [scaling-states scaling-state] (conj (or scaling-states []) scaling-state)))`

Reply:
Changed to:
`(swap! service-id->scaling-states-atom update service-id (fnil conj []) scaling-state)`